python/modules/map_plate.py

```diff
@@ -1,69 +1,104 @@
 #! /usr/bin/python
-import string, os, logging
+import string, os, logging, csv
+from collections import OrderedDict
 import modules.file_managment as FM
 import modules.csv_managment as CSV_M
-import csv
 
 logger = logging.getLogger("map plate")
 logger.info("Executing map_plate module.")
+fatal_error_msg = "Fatal error occured while trying to read mapplate."
 
 def parse_mp(input_path, delimiter):
+    """
+    Main function of TASK_READ_MAP_PLATE. 
+    Parse all map_plate informations (from csv files) into python dictionary.
+    Each well is represented by dictionary key, 
+    which value is a dictionary of all experimental settings.
+    """
     mp_dict = {}
     dim =_parse_base_params(input_path, delimiter, mp_dict)
     _parse_params(input_path, delimiter, dim, mp_dict)
     return mp_dict
 
 def _parse_params(input_path, delimiter, dimensions, mp_dict):
+    """
+    Params are information about experiment collected in csv files:
+    i.e. cells type, stimulation time.
+    """
```

Comment:
It was changed earlier to:

>    Parse information about experiment collected in csv files, i.e.:
>     - cells type
>     - stimulation time
>     - <...>
I can add params definition. However, I prefer to start comment with "parse" due to we describe most of function in such a way (function gets/parse/extracts/does etc)